src/Core/Core.php

```diff
@@ -70,20 +71,26 @@
 $requestURL = isset($_REQUEST['url']) ? trim($_REQUEST['url'], '/') : false;
 $flush = (isset($_GET['flush']) || $requestURL === trim(BASE_URL . '/dev/build', '/'));
 
-global $manifest;
-$manifest = new ClassManifest(BASE_PATH, false, $flush);
+// Manifest cache factory
+$manifestCacheFactory = new ManifestCacheFactory([
+    'namespace' => 'manifestcache',
+    'directory' => getTempFolder(),
```

Comment:
It's just a default argument; You can ignore it if you are using an in-memory cache (set via SS_MANIFESTCACHE).